### PR TITLE
Support babel's _interopRequireDefault for inlining fs.readFileSync

### PR DESCRIPTION
**Pull request by d4rkr00t**
Closes: #106 

This PR fixes an issue with babel transpiled import for fs.readFileSync:
```js
var _fs = require("fs");

var _fs2 = _interopRequireDefault(_fs);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var name = _fs2.default.readFileSync(__driname + '/name.text');
document.body.innerHTML = "Hello, " + name + "! Inlining via fs.readFileSync is working!";
```

It sort of simplifies `referencesImport` method, but at the same time it's not the "most performant" solution (traverse couple of times) :( I ran tests several times and couldn't see any performance regressions, but they are too small to be able to reveal any perf impact.

Also i re-enabled fs-import test. It wasn't included for, i guess, the same reason.

Another option could be detecting babel presets/plugins that transpile imports and show error message.

### CODE REVIEW

**Comment by devongovett** [comment]
hmm, perhaps we should just turn off babel transpiling modules. we could automatically set the `modules: false` option to babel-preset-env etc. pre dependency detection. we already transpile only modules if needed post detection, so this shouldn't break anything. thoughts?

**Comment by d4rkr00t** [comment]
Agree, it shouldn't break anything. i'll update my PR soonish, thanks :)

File: src/transforms/babel.js [hunk 1]
```diff
@@ -8,22 +8,47 @@ module.exports = async function(asset) {
...
+    (await config.load(asset.name, ['.babelrc', '.babelrc.js'])) || {};
+  let babelConfig = Object.assign(babelrc, {
     code: false,
-    filename: asset.name
-  };
+    filename: asset.name,
+    babelrc: false
+  });
+
+  // TODO: Support .babelignore
```
Comment (d4rkr00t) [inline src/transforms/babel.js]: In order to set modules: false to babel-preset-env I had to disabled automatic loading of .babelrc which as well disables loading of .babelrcignore :(

Comment (d4rkr00t) [reply src/transforms/babel.js]: I'll add support for that if you are happy with everything else

**Comment by d4rkr00t** [comment]
### Updates:

Instead of changing `referencesImport` logic I make babel transform load `.babelrc`, update opts for presets and use it instead of babel looking up configs on its own.

**Comment by devongovett** [comment]
This appears to have been fixed by other changes.

**Pull request closed without merging**